apps/condo/domains/user/components/CookieAgreement.tsx

```diff
@@ -0,0 +1,94 @@
+import styled from '@emotion/styled'
+import cookie from 'js-cookie'
+import React, { useState } from 'react'
+
+import { ChevronDown } from '@open-condo/icons'
+import { useIntl } from '@open-condo/next/intl'
+import { Typography, Button, Space } from '@open-condo/ui'
+
+import { useContainerSize } from '@condo/domains/common/hooks/useContainerSize'
+
+const COOKIE_AGREEMENT_KEY = 'cookieAgreementAccepted'
+
+const StyledFixedPanel = styled.div`
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  box-shadow: 0 14px 34px rgba(208, 216, 225, 0.4), 0 15px 38px rgba(208, 216, 225, 0.4), 0 7px 17px rgba(208, 216, 225, 0.6);
```

Comment:
Seems to me this shadow already exists in constants. Will be better to use it from there